util.go

```diff
@@ -69,7 +69,9 @@ func pinInfoSliceToSerial(pi []api.PinInfo) []api.PinInfoSerial {
 	return pis
 }
 
-func globalPinInfoSliceToSerial(gpi []api.GlobalPinInfo) []api.GlobalPinInfoSerial {
+// GlobalPinInfoSliceToSerial is a helper function for serializing a slice of
+// api.GlobalPinInfos.
+func GlobalPinInfoSliceToSerial(gpi []api.GlobalPinInfo) []api.GlobalPinInfoSerial {
```

Comment:
It does not need exporting, does it ? 

If exported, it would fit better as part of `api`.